specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json

```diff
@@ -734,6 +734,164 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "get": {
+        "tags": [
+          "PrivateEndpointConnections"
+        ],
+        "operationId": "PrivateEndpointConnections_Get",
+        "description": "Gets the specified private endpoint connection associated with the storage account.",
+        "x-ms-examples": {
+          "StorageAccountGetPrivateEndpointConnection": {
+            "$ref": "./examples/StorageAccountGetPrivateEndpointConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/StorageAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/PrivateEndpointConnectionName"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
Please add a default response that implements the common error contract. This should be on all API responses, but we can start with net new ones. Here is an example: https://github.com/Azure/azure-rest-api-specs/blob/14464cf0bec27f838f1649f562c9f98cc4074855/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105

Reply:
> Please add a default response that implements the common error contract. This should be on all API responses, but we can start with net new ones. Here is an example:
> 
> https://github.com/Azure/azure-rest-api-specs/blob/14464cf0bec27f838f1649f562c9f98cc4074855/specification/batch/resource-manager/Microsoft.Batch/stable/2019-08-01/BatchManagement.json#L105

Added default error response for these new apis.